cmake/sca/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Static Code Analysis (SCA) options
+
+# Copyright (c) 2023 Baumer (www.baumer.com)
```

Comment:
```suggestion
# Copyright (c) 2024 Baumer (www.baumer.com)
```

Reply:
fixed. Seems like I still living in 2023 😄